src/receipt/_style.scss

```diff
@@ -0,0 +1,67 @@
+#receipt-container {
+    width: 1170px;
+    padding-top: 2rem;
+}
+
+.page__receipt {
+    h2 {
+        margin-bottom: 10px;
+        font-weight: 400;
+    }
+
+    .thank-you {
+        font-size: 38px;
+    }
+
+    .info-item {
+        overflow: hidden;
+    }
+
+    .copy {
+        > div {
+            margin-bottom: 1rem;
+            line-height: 1.5rem;
+        }
+
+        .billing-address {
+            margin: 2rem 0;
+        }
+    }
+
+    .credit-messaging {
+        margin-top: 2rem;
+    }
+
+    .enterprise-message {
+        margin-bottom: 2rem;
+    }
+
+    .order-total {
+        font-size: 0.875rem;
+        margin-top: 3rem;
+
+        .order-total-item {
+            padding: 1rem;
+            font-weight: bold;
+        }
+
+        .border-divider {
+            border-bottom: 1px solid $gray-200;
+        }
+
+        .enterprise-customer {
+            padding-bottom: 12px;
+            color: $gray-300;
+            text-align: center;
+        }
+    }
+
+    #cta-nav-links {
+        margin: 2rem 0 4rem 0;
+
+        .dashboard-link {
+            margin-right: 2.5rem;
```

Comment:
Related to other feedback, if we treat these links at button links instead of `Hyperlinks`, I believe  `ActionRow` should handle the spacing between the two action buttons already without needing explicit custom spacing.

Example (see default spacing between action buttons in `ActionRow`):
<img width="219" alt="image" src="https://user-images.githubusercontent.com/2828721/186754972-ca013191-9521-4fbf-8dbd-305c4d89d8be.png">

Reply:
Makes sense now that I understand `ActionRow` better! Someone in fedex recommended this and I didn't stop to think that it needs an action (for the other case I used for just text formatting)